partner_affiliate/tests/test_partner_affiliate.py

```diff
@@ -0,0 +1,84 @@
+from odoo.tests import common
+
+
+class TestPartnerAffiliate(common.TransactionCase):
+
+    def setUp(self):
+        super(TestPartnerAffiliate, self).setUp()
+
+        self.first_parent = self.env['res.partner'].create({
+            'name': 'MyFirstParentForTheAffiliate',
+            'type': 'contact',
+            'is_company': True,
+            'street': 'first parent street',
+            'street2': 'number 99',
+            'zip': 123,
+            'city': 'Test City',
+        })
+
+        self.second_parent = self.env['res.partner'].create({
```

Comment:
You could reuse self.env['res.partner']

Reply:
Even you could define it to be reused inside the tests by self.partner_obj = self.env['res.partner']

Reply:
Nice to know, thanks for the tip! Changed.